test/mocha/10-keystore.js

```diff
@@ -385,6 +385,70 @@ describe('bedrock-kms-http API', () => {
         should.exist(result);
         result.should.eql(expectedConfig);
       });
+      it('throw internal server error when attempting to update "meterId"',
+        async () => {
+          // TODO: Remove or update this test when the following is fixed
+          // eslint-disable-next-line max-len
+          // https://github.com/digitalbazaar/bedrock-kms-http/blob/main/lib/http.js#L182
+          const secret = '69ae7dc3-1d6d-4ff9-9cc0-c07b43d2006b';
+          const handle = 'testKeyUpdate';
+          const capabilityAgent = await CapabilityAgent.fromSecret(
+            {secret, handle});
+
+          const secret2 = 'ac36ef8e-560b-4f6c-a454-6bfcb4e31a76';
+          const handle2 = 'testKeyUpdate2';
+          const capabilityAgent2 = await CapabilityAgent.fromSecret(
+            {secret: secret2, handle: handle2});
```

Comment:
Is a second agent needed here to simplify? Could this test use just one -- isn't it focused on changing the meter ID, not on *also* changing the controller ID?

Reply:
Yeah makes sense. I've removed it and updated the test to use ezcap library f764988